src/bp-forums/functions.php

```diff
@@ -1149,3 +1149,35 @@ function bb_forums_link_preview_parse_url() {
 	add_action( 'wp_ajax_nopriv_bb_forums_parse_url', 'bb_forums_link_preview_parse_url' );
 }
 
+/**
+ * Backup the last group forum associations.
+ *
+ * @since BuddyBoss [BBVERSION]
+ *
+ * @param int $group_id Group ID.
+ * @param int $forum_id Forum ID.
+ */
+function bb_update_last_group_forum_associations( $group_id = 0, $forum_id = 0 ) {
+	if ( empty( $forum_id ) ) {
+		return;
+	}
+	$forum_id = bbp_get_forum_id( $forum_id );
+
+	// Use current group if none is set.
+	if ( empty( $group_id ) ) {
+		$group_id = bp_get_current_group_id();
+	}
+
+	// Get the values to backup.
+	$group_ids = array_filter( bbp_get_forum_group_ids( $forum_id ) );
+	$forum_ids = array_filter( bbp_get_group_forum_ids( $group_id ) );
+
+	if (
+		in_array( $group_id, $group_ids, true ) &&
+		in_array( $forum_id, $forum_ids, true )
+	) {
+		// Save the backups.
+		update_post_meta( $forum_id, '_last_bbp_group_ids', $group_ids );
+		groups_update_groupmeta( $group_id, 'last_forum_id', $forum_ids );
```

Comment:
```suggestion
		groups_update_groupmeta( $group_id, 'last_forum_id', $forum_id );
```

@yudhisthirnahar This should be singular and store an `int` value instead of an `array` if there is only one value.